cpu/stm32/include/periph/f4/periph_cpu.h

```diff
@@ -23,6 +23,12 @@
 extern "C" {
 #endif
 
+/**
+ * @brief   Starting address of the ROM bootloader
+ *          see application note AN2606
+ */
+#define STM32_LOADER_ADDR   (0x1FFF0000)
```

Comment:
For STM32F312xx/423xxx it is `0x1FF00000` instead.

Reply:
Thank you!